network/dnsimple.py

```diff
@@ -130,12 +130,13 @@
 '''
 
 import os
+from sys import exit
 try:
     from dnsimple import DNSimple
     from dnsimple.dnsimple import DNSimpleException
 except ImportError:
     print "failed=True msg='dnsimple required for this module'"
-    sys.exit(1)
+    exit(1)
```

Comment:
actually, this should not be an exit at all, it should set a var so later in main we can exist with fail_json